source/2016-01-12-spec-for-has_many-with-dependent-option.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: Spec for has_many with dependent option
+date: 2016-01-12 14:43 UTC
+author: lex
+tags:
+---
+
+Every rails app almost certainly contains one or more `has_many` associations.
+Omitting the `dependent` option means that on deletion the foreign key gets
+nullified. In many cases this is not the expected behaviour though. And even if
+it is, it makes sense that we think about the behaviour of the relation for
+each `has_many` definition. Not thinking about how the associated objects
+behave on deletion often results in orphaned database entries and even in
```

Comment:
"and even" -> "or even" sounds better for me, but maybe that's just because it's phrased like this in German.